foreman-selinux-enable

```diff
@@ -13,15 +13,22 @@ trap "rm -rf '$TMP'" EXIT INT TERM
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    # UPGRADE STEP for Foreman 1.8 (remove after release)
+    # If katello policy was not yet loaded and elasticsearch port is still defined,
+    # remove it (it was moved to katello policy). Katello policy will not load before
+    # we remove the port definition (because SELinux prevents that).
+    semodule -l | grep -q katello || \
```

Comment:
It might work, but we'd have to enable it after too?  (as katello-selinux may not be upgraded)